tests/apps/example/templates/payment_intent.html

```diff
@@ -152,7 +152,7 @@ <h1 class="is-size-3">Example Payment Intent Manual Configuration</h1>
 	  } else if (response.requires_action) {
 		// Use Stripe.js to handle required card action
 		stripe.handleCardAction(
-		  response.payment_intent_client_secretemail
```

Comment:
FYI @tiwariayush

Reply:
Ah thanks for noticing. I didn't check the implementation of example with other cards than the success one (4242).

Reply:
Note that it still fails at the next step since it's trying to call `ajax/confirm_payment`

Reply:
Ah yes, I'm fixing the template currently. `ajax/confirm_payment` should point to the same view `example/payment-intent`

Reply:
Fixed in cc9bfa228